e2e/playwright/copy-move-actions/src/tests/move.test.ts

```diff
@@ -68,6 +69,10 @@ test.describe('Move actions', () => {
     await personalFiles.navigate();
   });
 
+  test.afterAll(async () => {
```

Comment:
can you please check/remove afterAll written at line 42

Reply:
Oh I didnt see its already there...